modules/auxiliary/scanner/oracle/tnspoison_checker.rb

```diff
@@ -42,8 +42,19 @@ def run_host(ip)
       send_packet = tns_packet("(CONNECT_DATA=(COMMAND=service_register_NSGR))")
       sock.put(send_packet)
       packet = sock.read(100)
+      hex_packet = Rex::Text.to_hex(packet, prefix = ':')
```

Comment:
when i set `RPORT = 8080`, i get a stack trace on line 45 because `packet` comes back `nil`.  I verified this behavior was happening before your changes, but the way the logic was set up swallowed the `nil` and reported incorrectly 'is vulnerable'.